README.md

```diff
@@ -95,6 +95,7 @@ _To be written_
 
 [Node.js]: https://github.com/nodejs/node
 [`npm` package ecosystem]: https://www.npmjs.com
+[Join the Discord channel]: https://discord.gg/hCgptwH
```

Comment:
Is this a temporary discord link? I always forget how to tell but I know there are temporary and permanent ones.

Reply:
This is set to "never expire"